eZ/Publish/Core/Persistence/Cache/LocationHandler.php

```diff
@@ -120,15 +144,19 @@ public function loadParentLocationsForDraftContent($contentId)
     /**
      * {@inheritdoc}
      */
-    public function loadByRemoteId($remoteId)
+    public function loadByRemoteId($remoteId, array $translations = null, bool $useAlwaysAvailable = true)
     {
-        $cacheItem = $this->cache->getItem("ez-location-${remoteId}-by-remoteid");
+        $translationsKey = $this->getCacheTranslationKey($translations, $useAlwaysAvailable);
+        $cacheItem = $this->cache->getItem("ez-location-remoteid-${remoteId}-${translationsKey}");
         if ($cacheItem->isHit()) {
             return $cacheItem->get();
         }
 
-        $this->logger->logCall(__METHOD__, array('location' => $remoteId));
-        $location = $this->persistenceHandler->locationHandler()->loadByRemoteId($remoteId);
+        $this->logger->logCall(
+            __CLASS__ . '::loadList',
```

Comment:
```suggestion
            __CLASS__ . '::loadByRemoteId',
```

Reply:
This should actually be `__METHOD__`, I'll fix :)